atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellLoader.java

```diff
@@ -123,11 +140,9 @@ void loadWithTs(String kvsMethodName,
             final CassandraKeyValueServices.ThreadSafeResultVisitor visitor,
             final ConsistencyLevel consistency) {
         final ColumnParent colFam = new ColumnParent(CassandraKeyValueServiceImpl.internalTableName(tableRef));
-        CellLoadingBatcher batcher = CellLoadingBatcher.create(
-                (numRows) -> logRebatchingWarnMessage(host, tableRef, numRows));
-
         List<Callable<Void>> tasks = Lists.newArrayList();
-        for (final List<Cell> partition : batcher.partitionIntoBatches(cells)) {
+        for (final List<Cell> partition : batcher.partitionIntoBatches(
+                cells, (numRows) -> logRebatchingWarnMessage(host, tableRef, numRows))) {
```

Comment:
You can pass in the Consumer into the CLB constructor, and then have partitionIntoBatches take the host and tableRef as arguments

Reply:
Sure, that's fine - have changed that. I was intending for this to be reused elsewhere (conceptually `InetSocketAddress` and `TableReference` aren't exactly things which a batcher should know about) but this should be fine for now.